src/uwtools/tests/drivers/test_mpas_init.py

```diff
@@ -0,0 +1,233 @@
+)
+def test_MPASInit_files_copied(config, cycle, key, task, test, tmp_path):
+    atm, sfc = "gfs.t%sz.atmanl.nc", "gfs.t%sz.sfcanl.nc"
+    atm_cfg_dst, sfc_cfg_dst = [x % "{{ cycle.strftime('%H') }}" for x in [atm, sfc]]
+    atm_cfg_src, sfc_cfg_src = [str(tmp_path / (x + ".in")) for x in [atm_cfg_dst, sfc_cfg_dst]]
+    config["mpas_init"].update({key: {atm_cfg_dst: atm_cfg_src, sfc_cfg_dst: sfc_cfg_src}})
+    path = tmp_path / "config.yaml"
+    with open(path, "w", encoding="utf-8") as f:
+        yaml.dump(config, f)
+    driverobj = mpas_init.MPASInit(config=path, cycle=cycle, batch=True)
+    atm_dst, sfc_dst = [tmp_path / (x % cycle.strftime("%H")) for x in [atm, sfc]]
+    assert not any(dst.is_file() for dst in [atm_dst, sfc_dst])
+    atm_src, sfc_src = [Path(str(x) + ".in") for x in [atm_dst, sfc_dst]]
+    for src in (atm_src, sfc_src):
+        src.touch()
+    getattr(driverobj, task)()
+    assert all(getattr(dst, test)() for dst in [atm_dst, sfc_dst])
+
+
+def test_MPASInit_namelist_file(driverobj):
+    dst = driverobj._rundir / "namelist.init_atmosphere"
+    assert not dst.is_file()
+    driverobj.namelist_file()
+    assert dst.is_file()
+    assert isinstance(f90nml.read(dst), f90nml.Namelist)
+
+
+def test_MPASInit_provisioned_run_directory(driverobj):
+    with patch.multiple(
+        driverobj,
+        boundary_files=D,
+        files_copied=D,
+        files_linked=D,
+        namelist_file=D,
+        runscript=D,
+        streams_file=D,
+    ) as mocks:
+        driverobj.provisioned_run_directory()
+    for m in mocks:
+        mocks[m].assert_called_once_with()
+
+
+def test_MPASInit_run_batch(driverobj):
+    with patch.object(driverobj, "_run_via_batch_submission") as func:
+        driverobj.run()
+    func.assert_called_once_with()
+
+
+def test_MPASInit_run_local(driverobj):
+    driverobj._batch = False
+    with patch.object(driverobj, "_run_via_local_execution") as func:
+        driverobj.run()
+    func.assert_called_once_with()
+
+
+def test_MPASInit_runscript(driverobj):
+    dst = driverobj._rundir / "runscript.mpas_init"
+    assert not dst.is_file()
+    driverobj._driver_config["execution"].update(
+        {
+            "batchargs": {"walltime": "00:01:00"},
+            "envcmds": ["cmd1", "cmd2"],
+            "mpicmd": "runit",
+            "threads": 8,
+        }
+    )
+    driverobj._config["platform"] = {"account": "me", "scheduler": "slurm"}
+    driverobj.runscript()
+    with open(dst, "r", encoding="utf-8") as f:
+        lines = f.read().split("\n")
+    # Check directives:
+    assert "#SBATCH --account=me" in lines
+    assert "#SBATCH --time=00:01:00" in lines
+    # Check environment commands:
+    assert "cmd1" in lines
+    assert "cmd2" in lines
+    # Check execution:
+    assert "test $? -eq 0 && touch %s/done" % driverobj._rundir
+
+
+def test_MPASInit_streams_file(driverobj):
```

Comment:
MPAS naming for test --> `def test_MPAS_streams(driverobj):`,  remove `file`

Reply:
I'm going to keep `_file` in the test name to keep it consistent with `test_MPAS_namelist_file()` and the function names in `drivers/mpas_init.py`.